lib/restful_resource/http_client.rb

```diff
@@ -240,7 +237,7 @@ def initialize_connection(logger: nil,
         b.response :encoding
         b.use :gzip
 
-        b.adapter :typhoeus
+        b.adapter :net_http
```

Comment:
I'd actually we'd rather do ⬇️ and allow services to set this themselves.

```suggestion
        b.adapter Faraday.default_adapter
```

Reply:
Ah, of course 👍